manifests/init.pp

```diff
@@ -72,6 +72,7 @@
   $qpid_router_broker_port   = $capsule::params::qpid_router_broker_port,
 ) inherits capsule::params {
 
+  include ::apache
```

Comment:
I agree that this has become much more of a profile module and we complete that transition.